lib/src/adder.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// carry_save_multiplier.dart
```

Comment:
Check the names of files and file descriptions, copy-paste error in multiple files.